YvesIraguha/lost-and-found-api
# #171926509 Enabling signIn with google

#### What does this PR do?
* Sign in with google
#### Description of Task to be completed?
* Enabling Google auth using passport.js
#### How should this be manually tested?
* Clone the repo and run `cd lostAndFound.Api`
* Checkout the branch; `git checkout ft-enable-social-auth-#171926509`
* Update your `.env` file with `.env.sample` data
* Then run `npm install` to install all dependencies
* Then run `npm run dev` to start the server
* In your preferred browser go to `http://localhost:3000/auth/google`
*After redirecting and sign in with your google account, you should see*
** token and its value
** user with his saved data
#### Any background context you want to provide?
data to fill with CLIENT_ID and CLIENT_SECRET in `.env` file are found on slack 
#### What are the relevant pivotal tracker stories?
#171926509
